components/ingest-service/test/chef_action.js

```diff
@@ -65,25 +65,6 @@ describe("/events/chef/action", function () {
       });
     });
 
-    describe("actions with user updates are persisted", function () {
```

Comment:
The data is persisted in the event-feed-service and config-mgmt-service's PostgreSQL, so it can not be tested here.